samples/bluetooth/st_ble_sensor/README.rst

```diff
@@ -0,0 +1,37 @@
+.. _bluetooth-st_ble_sensor:
+
+Bluetooth: ST BLE Sensor Demo
+#############################
+
+Overview
+********
+
+This application demonstrates BLE peripherals on an `ST Nucleo WB55RG`_
+by exposing vendor-specific GATT services.
+Currently only button notification and LED service are implemented.
+
+Requirements
+************
+
+* `ST BLE Sensor Android app <ST BLE Sensor app>`_
+* A board with STM32WB55 like ST Nucleo WB55RG
+* This application requests having the stm32wb5x_BLE_Stack_fw.bin binary flashed on the Wireless Coprocessor.
+You need to use <STM32CubeProgrammer>`_ to load the appropriate binary.
```

Comment:
Fix the reference link, and I'm assuming this is a new bullet point, so change to:
```
* Use `STM32CubeProgrammer`_ to load the appropriate binary.
```

Reply:
I moved 1 bullet to Overview and added STM32CubeProgrammer bullet to Requirements